gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
What I've been trying to say is, it _won't_. The jinja templating is server side, not build-time. If you look at `dist/1.0/index.html`, you'll see:

```
{% include 'templates/head-meta.html' %}
```

The `head-meta` template gets interpolated on the server at request time by `server.py`:

https://github.com/Polymer/docs/blob/master/server.py#L59